Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/package.mo

```diff
@@ -1,10 +1,9 @@
 within Buildings.Fluid.HeatExchangers;
-package AirToAirHeatRecovery "Package with Air-to-Air heat recovery devices"
+package AirToAirHeatRecovery "Package with air-to-air heat recovery devices"
   extends Modelica.Icons.VariantsPackage;
-
-  annotation (Documentation(info="<html>
+annotation (Documentation(info="<html>
 <p>
-This package contains components models for air-to-air energy recovery devices.
+This package contains component models for air-to-air energy recovery devices.
```

Comment:
change the `energy recovery devices` to `heat recovery devices`. It will be consistent throughout the entire package.